packages/gatsby-plugin-sharp/src/process-file.js

```diff
@@ -43,6 +44,18 @@ const argsWhitelist = [
   `background`,
 ]
 
+let cloudEventsEmitter
+if (process.env.GATSBY_CLOUD_IMAGE_SERVICE_URL) {
```

Comment:
I went with assumption that cloud worker doesn't have this env var set - otherwise `processFile` would be circularly calling itself I think